cmd/application.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+)
+
+var appCmd = &cobra.Command{
+	Use:     "app",
+	Aliases: []string{"apps, application, applications"},
+	Short:   "Manage Applications inside your Civo account",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := cmd.Help()
+		if err != nil {
+			return err
+		}
+		return errors.New("a valid subcommand is required")
+	},
+}
+
+var appDomainCmd = &cobra.Command{
+	Use:     "domain",
+	Aliases: []string{"domains"},
+	Short:   "Details of your application domains",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := cmd.Help()
+		if err != nil {
+			return err
+		}
+		return errors.New("a valid subcommand is required")
+	},
+}
+
+var appConfigCmd = &cobra.Command{
+	Use:     "config",
+	Aliases: []string{"conf"},
+	Short:   "Configure your application",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := cmd.Help()
+		if err != nil {
+			return err
+		}
+		return errors.New("a valid subcommand is required")
+	},
+}
+
```

Comment:
Nit: Can you move the scale subcommand to be a part of this file? The inner `Run` function can remain in a separate file but it'll be nice  to stick with the pattern fo initilasing all sub-commands from this file.

Reply:
This is done in the latest commit.